docs/upgrade-guide.md

```diff
@@ -2,14 +2,33 @@
 
 ## Migrating from react-map-gl
 
+### Update Imports
+
 `@vis.gl/react-maplibre` v1.0 can be used as a drop-in replacement of `react-map-gl` v7:
 
 ```patch
 - import {Map, Marker} from 'react-map-gl/maplibre';
 + import {Map, Marker} from '@vis.gl/react-maplibre';
 ```
 
+### Remove Mapbox-only Props
+
 The following Mapbox-only props from `react-map-gl`'s Map component are removed:
+
 - `mapboxAccessToken`
 - `workerClass`
 - `baseApiUrl`
+
+### Removed default for `RTLTextPlugin`
```

Comment:
```suggestion
### Explicitly set `RTLTextPlugin` (if needed)
```

Since we are in the "migrate from …"  section, this should read like a "do this…" headline, not a "what happened" headline.